alphadia/transferlearning/train.py

```diff
@@ -432,7 +436,9 @@ def finetune_ms2(
 
         # Shuffle the psm_df and split it into train and test
         train_psm_df = psm_df.sample(frac=self.settings["train_ratio"]).copy()
-        test_psm_df = psm_df.drop(train_psm_df.index).copy()
+        rest_psm_df = psm_df.drop(train_psm_df.index).copy()
+        val_psm_df = rest_psm_df.sample(frac=self.settings["validation_ratio"]).copy()
```

Comment:
What do you think about adding a logger info with `test_ratio` and `n_test` for all three subsets?